lib/util/Makefile

```diff
@@ -5,13 +5,17 @@ LIB_DIR ?= ..
 include $(LIB_DIR)/defines.mk
 include $(LIBXDP_DIR)/libxdp.mk
 
-all: $(UTIL_OBJS)
+UTIL_OBJS_ALL := $(UTIL_OBJS) xdp_sample.o
+
+all: $(UTIL_OBJS_ALL)
 
 # Create expansions for dependencies
 UTIL_H := ${UTIL_OBJS:.o=.h}
 
-$(UTIL_OBJS): %.o: %.c $(UTIL_H) $(LIBMK)
+xdp_sample.o: ../../headers/xdp_sample.h ../../headers/xdp_sample_shared.h
```

Comment:
`headers/` is for header files we install; if they're internal, just put them in lib/util along with the .c and don't make this adjustment...